crtauth/ssh.py

```diff
@@ -22,10 +22,14 @@
 import os
 import socket
 import struct
+import six
 
 from crtauth import exceptions
 from crtauth import rsa
 
+from six.moves import range
```

Comment:
My understanding of this import is that changes python 3 range semantics to match the python 2 ones. Since we already opted for the python 3 semantics with xrange I guess this import could go away.